lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -36,14 +29,14 @@ def self.run(params)
         Helper::RevenuecatInternalHelper.edit_changelog(generated_contents, changelog_latest_path, editor)
         changelog = File.read(changelog_latest_path)
 
-        Helper::RevenuecatInternalHelper.create_new_release_branch(new_version_number)
+        Helper::RevenuecatInternalHelper.create_and_checkout_new_branch("release/#{new_version_number}")
         Helper::RevenuecatInternalHelper.replace_version_number(version_number,
                                                                 new_version_number,
                                                                 files_to_update,
                                                                 files_to_update_without_prerelease_modifiers)
         Helper::RevenuecatInternalHelper.attach_changelog_to_master(new_version_number, changelog_latest_path, changelog_path)
         Helper::RevenuecatInternalHelper.commmit_changes_and_push_current_branch("Version bump for #{new_version_number}")
-        Helper::RevenuecatInternalHelper.create_release_pr(new_version_number, changelog, repo_name, github_pr_token)
+        Helper::RevenuecatInternalHelper.create_pr_to_main("Release/#{new_version_number}", changelog, repo_name, github_pr_token)
```

Comment:
we don't have any rules for ruby code yet, but I wonder if we should start using named params. 
It took me a second to understand that the reason that on line 32 we do `release/` (lowercase) and then on 39 `Release/` is that one is the branch name and the other one the PR title

Reply:
Hmm it's a good point... I'm a bit reticent doing it for methods with few parameters, instead we can store the parameter in a properly named variable (as I will do in this case) but I totally agree for methods with a lot of parameters. In this case with 4 parameters, I'm a bit on the fence... I would say to do it with 5+ parameters but I would be happy to discuss a different number. Lmk what you think!